net.poedit.Poedit.json

```diff
@@ -0,0 +1,99 @@
+{
+    "app-id": "net.poedit.Poedit",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "poedit",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=home",
+        /* Network */ 
+        "--share=network",
+        /* Dconf */
+        "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "rename-appdata-file": "poedit.appdata.xml",
+    "rename-desktop-file": "poedit.desktop",
+    "rename-icon": "poedit",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "modules": [{
+            "name": "wxWidgets",
+            "rm-configure": true,
+            "config-opts": [
+                "--with-gtk=3", "--disable-sdltest", "--disable-webkit", "--disable-webview",
+                "--disable-webviewwebkit"
+            ],
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://github.com/wxWidgets/wxWidgets/releases/download/v3.1.0/wxWidgets-3.1.0.tar.bz2",
```

Comment:
Using wx-3.1.0, a year-old *unstable development snapshot*, is a very bad idea — it is known buggy. Reasonable choices are latest stable or the submodule from Poedit's own tree.

Reply:
Should I switch to 3.0.3?

Reply:
Plus would like to hear your opinion about including cld2 & c++ rest sdk

Reply:
The warning you quoted is pretty clear on my opinion, I would think: packaging degraded versions is frowned upon.